releasing-archway.md

```diff
@@ -0,0 +1,111 @@
+  - [Patch releases](#patch-releases) are released as we cherrypick commits from main into
+    the release-X.Y branch, as needed.
+  - X.Y.Z is cut straight from the release-X.Y branch, and X.Y.Z+1-beta.0 is
+    tagged on the followup commit that updates pkg/version/base.go with the beta
+    version.
+- X.Y.Z, Z > 0 (Branch: release-X.Y.Z) (Branched [patch releases](#patch-releases) only for hotfix situations)
+  - These are rarely used and are special in that the X.Y.Z tag is branched to isolate
+    the emergency/critical fix from all other changes that have landed on the
+    release branch since the previous tag
+  - Cut release-X.Y.Z branch to hold the isolated patch release
+  - Tag release-X.Y.Z branch + fixes with X.Y.(Z+1)
+  - Branched [patch releases](#patch-releases) are rarely needed but used for
+    emergency/critical fixes to the latest release
+
+### Major version timeline
+
+There is currently no mandated timeline for major versions beyond version 1.Y.Z, Y,Z >= 0. We haven't so far applied a rigorous interpretation of semantic
+versioning with respect to incompatible changes of any kind.
+
+TODO: Major versioning criteria need to be put up for discussion once Archway protocol reaches 1.Y.Z release.
+
+## Patch releases
+
+Patch releases are intended for critical bug fixes to the latest minor version,
+such as addressing security vulnerabilities, fixes to problems affecting a large
+number of users and severe problems with no workaround.
+
+They should not contain miscellaneous feature additions or improvements, and
+especially no incompatibilities should be introduced between patch versions of
+the same minor version (or even major version).
+
+Dependencies, such as cosmos-sdk or tendermint, should also not be changed unless
+absolutely necessary, and also just to fix critical bugs (so, at most patch
+version changes, not new major nor minor versions).
+
+## Network Upgrades
+
+- Upgrade documentation and a graceful migration path must be provided between all **consective** minor releases.
+  e.g. going from v1.3.Z, Z >= 0 to v1.4.Z, Z >= 0 must be accompanied by relevant documentation and automated migration path.
+- Network upgrade must also follow sequential minor releases. e.g. if a network is running v1.2.Z, Z >= 0, it must first
+  upgrade to v1.3.Z, Z >= 0 before upgrading to v1.4.Z, Z >= 0
+- Network upgrades may skip patch release e.g. a network running v1.2.1 may directly upgrade to v1.2.4
+- Network upgrades on archway-1 and constantine-1 must happen through software upgrade proposals.
+- Once a software upgrade proposal passes, all network operator are expected to upgrade to the exact
+  same archway protocol release within a reasonable timeframe.
+- Each new release must be adopted and tested on constantine-1 before being adopted on archway-1.
+- titus-1 is an unstable development network, with a single validator. Network upgrades on titus-1 are
```

Comment:
We probably want to expand Titus to be running at least 3 nodes;
A single node testnet will fail to detect any non-determinism errors as these typically manifest as consensus errors, which can only occur if the network is running more than a single node;

_Three nodes are statistically more likely to experience consensus errors than would two nodes (which significantly improves testing time)_

Reply:
Agreed will remove the 'single validator' here